src/components/split-button/split-button.test.js

```diff
@@ -184,8 +163,8 @@ context("Tests for Split Button component", () => {
     });
   });
 
-  describe("pressing tab while SplitButton is open", () => {
-    it("should move focus to next child button and to second SplitButton when end of list reached", () => {
+  describe("pressing tab while Split Button is open", () => {
```

Comment:
I notice you've changed `SplitButton` to `Split Button` here and in many places - presumably because we were inconsistent. I'm all for consistency but I wonder if `SplitButton` should be the preferred name, as that's the actual component name.

Reply:
Yeah, I changed that for consistency and pondered which way it should be done. I checked other multi-worded components and many had spaces in the description so I went with that, but on reflection it does make sense to make it the same as the component.

Reply:
I agree with @robinzigmond , the original name of the component should be used like `SplitButton`, could you amend that @sianford ?

Reply:
Already done, I just hadn't pushed the changes. I was waiting for confirmation of those hover states ;)